test/helpers/test-bin.js

```diff
@@ -36,13 +36,25 @@ function runWebackDevServer(testArgs, configPath) {
     child.on('error', (error) => reject(error));
 
     child.stdout.on('data', (data) => {
-      stdout += data.toString();
+      const str = data.toString();
+
+      stdout += str;
+
+      // if webpack.config.js is a promise function, it won't be able to call `close`
```

Comment:
I don't know why it couldn't call `close`, but I checked `examples/general/config-promise` and it works.